### PR TITLE
Pm 2019 09 email templates

### PR DESCRIPTION
**Pull request by phillimorland**
Currently cannot test due to local background tasks error (trying to fix). Have added notation where I don't know how to get the correct link/text from django.

### CODE REVIEW

File: meinberlin/apps/notifications/templates/meinberlin_notifications/emails/notify_creator.de.email [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
 {% block subject %}Neue Reaktion auf Ihren Beitrag auf {{ site.name }}{% endblock %}
 
-{% block headline %}{{ action.actor.username }} hat auf Ihren Beitrag reagiert{% endblock  %}
+{% block headline %}Reaktion auf Ihren Beitrag{% endblock  %}
 
-{% block content %}{{ action.actor.username }} hat auf Ihren Beitrag reagiert. Schauen Sie nach, was in Ihrem Beitrag aktuell passiert.{% endblock %}
+{% block greeting %}Hallo {{ receiver.username }},{% endblock %}
+
+{% block content %}{{ action.actor.username }} hat Ihren Beitrag kommentiert:
+{{ action.obj.text }}
```
Comment (phillimorland) [inline meinberlin/apps/notifications/templates/meinberlin_notifications/emails/notify_creator.de.email]: here we need the text of the comment

File: meinberlin/apps/notifications/templates/meinberlin_notifications/emails/notify_creator.de.email [hunk 2]
```diff
@@ -3,11 +3,15 @@
...
 {% block cta_label %}{% if action.target.get_absolute_url %}Beitrag anzeigen{% else %}Projekt anzeigen{% endif %}{% endblock %}
 
-{% block reason %}Diese E-Mail wurde an {{ receiver.email }} gesendet. Die E-Mail wurde an Sie gesendet, da Sie einen Beitrag in einem Projekt erstellt haben. Wenn Sie diese E-Mails nicht länger erhalten möchten, können Sie dem Projekt auf der Projektseite entfolgen oder Benachrichtigungen in Ihren Benutzerkonto Einstellungen komplett deaktivieren.{% endblock %}
+{% block reason %}Diese E-Mail wurde an {{ receiver.email }} gesendet. Sie haben die E-Mail erhalten, weil Sie oben genanntem Projekt folgen. Wenn Sie keine Benachrichtigungen mehr erhalten möchten, entfolgen Sie dem {% block project_url %}{{ email.get_host }}{{action.obj.module.get_absolute_url }}{% endblock %}{% block project_label %}Projekt{% endblock %} oder ändern Sie die Einstellungen zu Ihrem {% block account_url %}{{ email.get_host }}{{ receiver.account.get_absolute_url }}{% endblock %}{% block account_label %}Account{% endblock %}.{% endblock %}
```
Comment (phillimorland) [inline meinberlin/apps/notifications/templates/meinberlin_notifications/emails/notify_creator.de.email]: link to receivers account setting

File: meinberlin/apps/embed/templates/meinberlin_embed/embed.html [hunk 3]
```diff
@@ -6,7 +6,7 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title>Embedded meinBerlin</title>
-    <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
+    <link rel="shortcut icon" type="image/x-icon" href="{% static 'images/favicon.ico' %}?v=2" />
```
Comment (fuzzylogic2000) [inline meinberlin/apps/embed/templates/meinberlin_embed/embed.html]: @rmader This was part of the attempt to fix @phillimorland 's local email sending. Should it stay like that or go back to the old form?

Comment (rmader) [reply meinberlin/apps/embed/templates/meinberlin_embed/embed.html]: Uh that looks weird. Definitely should not stay like that :/

Comment (rmader) [reply meinberlin/apps/embed/templates/meinberlin_embed/embed.html]: Ups, sorry, I read it the wrong way around. It looks absolutely like it should be that way!

**Pull request merged**